hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -226,6 +226,24 @@
         - 'kubernetes-e2e-{suffix}'
 
 - project:
+    name: kubernetes-e2e-gce-1-2
+    trigger-job: 'kubernetes-build-1.2'
```

Comment:
No, I should remove the branch variable from the others. It used to be used to call the runner for that branch, back when they were all `e2e.sh`.